rusoto/core/src/signature.rs

```diff
@@ -330,13 +362,15 @@ fn skipped_headers(header: &str) -> bool {
     ["authorization", "content-length", "user-agent"].contains(&header)
 }
 
+/// Returns standardised URI
```

Comment:
http://docs.aws.amazon.com/general/latest/gr/sigv4-create-canonical-request.html Again here for AWS Docs.